.github/workflows/examples-k32w.yaml

```diff
@@ -37,7 +37,7 @@ jobs:
         if: github.actor != 'restyled-io[bot]'
 
         container:
-            image: connectedhomeip/chip-build-k32w:0.5.1
+            image: connectedhomeip/chip-build-k32w:latest
```

Comment:
Do we have an existing preference for whether we should be using latest images or explicit tags?  This is probably convenient, but seems like it will reduce repeatability for our CI.